packages/build/tests/secrets_scanning/tests.js

```diff
@@ -52,7 +52,10 @@ test('secrets scanning, should skip when secrets passed but SECRETS_SCAN_OMIT_PA
   const output = await new Fixture('./fixtures/src_scanning_omit_glob_path')
     .withFlags({ debug: false, explicitSecretKeys: 'ENV_VAR_1,ENV_VAR_2' })
     .runWithBuild()
-  t.snapshot(normalizeOutput(output))
+
+  t.false(normalizeOutput(output).includes('found value at line 1 in dist/safefile.js'))
+  t.false(normalizeOutput(output).includes('found value at line 1 in dist/static-files/safefile.js'))
+  t.false(normalizeOutput(output).includes('found value at line 1 in src/static-files/safefile.js'))
 })
```

Comment:
I'm not sure if this will actually work - but I think it's good idea to try to assert that we do find secrets in paths that are not omitted by glob pattern?
```suggestion
  t.true(normalizeOutput(output).includes('found value at line 1 in dist/static-files/notsafefile.js'))
  t.true(normalizeOutput(output).includes('found value at line 1 in src/static-files/notsafefile.js'))  
})
```

Reply:
I think that case is already covered by `should fail build when it finds secrets in the src and build output`

Reply:
That one doesn't test the glob pattern behavior (at least it doesn't seems so to me at first glance - maybe underlying internals have default glob pattern in which case I would be fine with not changing things here)